api/swagger.yml

```diff
@@ -478,6 +489,32 @@ components:
           description: short message explaining the error
           type: string
 
+    ObjectError:
+      type: object
+      required:
+        - status_code
+        - message
+      properties:
+        status_code:
+          type: integer
+          description: http status code associated with the path/operation
+        path:
+          type: string
+          description: path to the associated object
+        message:
+          type: string
+          description: short message explaining the error
```

Comment:
```suggestion
        path:
          type: string
          description: affected path
        status_code:
          type: integer
          description: HTTP status code associated for operation on path
        message:
          type: string
          description: short message explaining status_code
```

Reply:
This puts code and message together, and also changes wording slightly.

Reply:
Thanks